src/components/Buttons/Button/Button.jsx

```diff
@@ -1,17 +1,23 @@
 import PropTypes from 'prop-types';
 import noop from '../../../shared/noop';
+import style from './Button.module.css';
 
-export const Button = ({ children, onClick, ...restProps }) => (
-  <button type='button' onClick={onClick} {...restProps}>
-    {children}
-  </button>
-);
+export function Button({ title, onClick, type, ...restProps }) {
```

Comment:
You violate the main principle. You should not override default tags properties. in this case you should set type as 'button' by default and use at as parameter. You should use different field for this `type: 'typePrimary'`

Reply:
Yes, thanks. Fixed.
```suggestion
export function Button({ title, onClick, stylingType, ...restProps }) {
```